modules/phase_field/include/utils/EBSDAccessFunctors.h

```diff
@@ -21,8 +21,8 @@ class EBSDAccessFunctors
 public:
   /// Per element EBSD data point
   struct EBSDPointData {
-    Real phi1, phi, phi2, symmetry;
-    unsigned int grain, phase, op, global;
+    Real phi1, Phi, phi2;
```

Comment:
Am I detecting some hostility here? This is a nested struct which was
merged that way. I can certainly switch that over to the standards though.
I just managed to f up my network access for this morning.

On Thu, Feb 25, 2016, 7:42 AM Cody Permann <notifications@github.com> wrote:

> In modules/phase_field/include/utils/EBSDAccessFunctors.h
> <https://github.com/idaholab/moose/pull/6405#discussion_r54103694>:
>
> > @@ -21,8 +21,8 @@ class EBSDAccessFunctors
> >  public:
> >    /// Per element EBSD data point
> >    struct EBSDPointData {
> > -    Real phi1, phi, phi2, symmetry;
> > -    unsigned int grain, phase, op, global;
> > +    Real phi1, Phi, phi2;
>
> :barf: - This keeps getting uglier. Is there a reference you can put in
> here so that we can note why you are intentionally not following the code
> standards?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6405/files#r54103694>.
>